frontend/src/components/Calendar/calendar.css

```diff
@@ -0,0 +1,183 @@
+* {
+    font-family: 'Montserrat', sans-serif;
+    font-style: normal;
+}
```

Comment:
Making sure you understand that this will apply to EVERY thing. If thats known, move this to a global style sheet rather than a calendar.css style sheet